src/App.js

```diff
@@ -0,0 +1,29 @@
+import { BrowserRouter, Redirect, Route, Switch } from "react-router-dom";
+import Todo from "./pages/Todo/Todo";
+import Login from "./pages/Login/Login";
+import Landing from "./pages/Landing/Landing";
+import { useGetUser } from "./hooks";
+
+function App() {
+  const [{ user, isLoading, isError }, dispatch] = useGetUser();
+  console.log("USER", user);
+
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Route path="/todos">
+          {user ? <Todo user={user} dispatch={dispatch} /> : <Redirect to="/login" />}
```

Comment:
React router has a guard mechanism for protected routes, take a look at that one.

Reply:
Okay Ill do that